automation/Update-Order-Standard-Work.md

```diff
@@ -8,6 +8,19 @@ The following sequence of actions should be taken to successfully update the Dev
 >
 > If you create a new version of documentation in readme, such as by copying an existing version and renaming the version name, readme.io will create new page IDs and new category IDs. This means that some pre-work is needed for a new version of the documentation. 
```

Comment:
That part can be updated, as now we have a script that handles this `pre-work` -  right?